CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+## [1.14.0/0.39.0/0.8.0] - 2023-01-30
+
 ### Added
 
 - Add the new `go.opentelemetry.io/contrib/instrgen` package to provide auto-generated source code instrumentation. (#3068)
```

Comment:
This section should be moved to the `## Unreleased` section. The `intrgen` module is still ignored in `versions.yaml` and isn't being released here.